tests/constbuffer_array_splitter_ut/constbuffer_array_splitter_ut.c

```diff
@@ -1009,7 +1009,7 @@ TEST_FUNCTION(constbuffer_array_splitter_split_to_array_of_array_with_null_buffe
     ///arrange
 
     /// act
-    TARRAY(CONSTBUFFER_ARRAY_HANDLE) result = constbuffer_array_splitter_split_to_array_of_array(NULL, 42);
+    TARRAY(CONSTBUFFER_ARRAY_HANDLE) result = constbuffer_array_splitter_split_to_array_of_array(NULL, 42, IGNORED_ARG);
```

Comment:
>IGNORED_ARG [](http://example.com/codeflow?start=107&length=11)

`IGNORED_ARG` is used for mock expectations only. It's equivalent to writing `NULL` here though. We should pass a real pointer in these tests otherwise you are not isolating which parameter is NULL.

(For example, if the code only checked `split_buffer_arrays_count` for NULL but we forgot to check `buffers`, this test would pass but we have a bug).

Same comment for the next test

Reply:
done